packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -71,6 +71,16 @@ type TrackScreenDetailsTileProps = {
   isLineupLoading: boolean
 }
 
+const recordPlay = (id, play = true) => {
```

Comment:
just wondering if this could be used elsewhere?

Reply:
It can! CollectionScreenDetailsTile and Lineup would be good candidates, I'll make a follow up PR